packages/editor/src/store/effects.js

```diff
@@ -177,33 +65,14 @@ export default {
 
 		// Apply a template for new posts only, if exists.
 		const isNewPost = post.status === 'auto-draft';
-		const template = getTemplate( state );
 		if ( isNewPost && template ) {
 			blocks = synchronizeBlocksWithTemplate( blocks, template );
```

Comment:
`block-editor` internally tracks the template. Should it be the one responsible for syncing from blocks?